components/wifi/esp8266-driver/mbed_lib.json

```diff
@@ -21,6 +21,14 @@
             "help": "RESET pin for the modem, defaults to Not Connected",
             "value": null
         },
+        "rst-pin-polarity": {
+            "help": "Polarity of RESET pin for the modem. [0/1]",
+            "value": 0
+        },
+        "rst-assert-time": {
```

Comment:
What is the time unit here? I suggest to call this `rst-assert-time-us` or something similar and adjust to whatever unit is needed in the code, for example `ThisThread::sleep_for(MBED_CONF_ESP8266_RST_ASSERT_TIME_US/100);`

If this is in system ticks, we can call it `rst-assert-time-ticks`.
@VeijoPesonen , any thoughts on this?

Reply:
I would like to know the reason why this needs to made configurable.

Reply:
@VeijoPesonen `rst-pin-polarity` and `rst-assert-time` originally are to achieve what #11299 is at. Also restate on the top.

Reply:
@michalpasztamobica Change to `rst-assert-time-us` and adjust related code.
```
ThisThread::sleep_for(MBED_CONF_ESP8266_RST_ASSERT_TIME_US / 1000);
```